sample-apps/radiology/lib/infers/deepedit.py

```diff
@@ -87,6 +88,7 @@ def pre_transforms(self, data=None):
         if self.type == InferType.DEEPEDIT:
             t.extend(
                 [
+                    OrientationGuidanceMultipleLabelDeepEditd(label_names=self.labels),
```

Comment:
Hi @diazandr3s , I suggest to add a `ref_image` arg to match up with other transforms.
I did this change in model zoo's PR:
https://github.com/Project-MONAI/model-zoo/pull/501/files#diff-7d1d3c4d9a0b5c7d8e7025ee44af385a0fed3005a63ae4ce68b7f27e639ef7a3R9

Reply:
Good idea! Is this what you mean @yiheng-wang-nv?

Reply:
Yes, thanks!